docs/bestpractices.md

```diff
@@ -0,0 +1,7 @@
+# Best Practices
+
+This document outlines some best practices for using DIDs, Verifiable Credentials, and other concepts that relate to Wallet-SDK.
+
+## DIDs
+
+* Generate a new DID for each new credential requested by the user.
```

Comment:
The client is suggested to generate a new DID for each new credential they request  This is considered best practice so the DID cannot be used as a correlating identifier.

Reply:
Done